### ISSUE TITLE
pandas remove_unused_categories inplace parameter is deprecated

### ISSUE THREAD

**Issue by ballenspectric** [spectriclabs/elastic_datashader#39]
```
pandas_util.py:51: FutureWarning: The `inplace` parameter in pandas.Categorical.remove_unused_categories is deprecated and will be removed in a future version.
```

**Comment by ballenspectric** [comment]
Pandas 2.0 has been released, and the `inplace` parameter has been removed.  It should probably be removed from the elastic_datashader code.

**Comment by desean1625** [comment]
fixed 9928acd
